src/app/setup/github-oauth.component.scss

```diff
@@ -1,6 +1,7 @@
-@import 'patternfly-sass/assets/stylesheets/patternfly/color-variables';
+@import '../../../node_modules/patternfly-sass/assets/stylesheets/patternfly/color-variables';
```

Comment:
Does this need to be referenced as a relative path?  Maybe we can fix the tooling so that's not needed, or maybe we we can have a top-level scss file that imports all of this stuff so that we don't have to have this in every component scss file.

Reply:
This is an absolute pain, I'd love to not have to use relative paths. We can open up an issue for it after TP1.

Reply:
I think webpack can help with that